packages/rune-games-cli/src/flows/Create.tsx

```diff
@@ -175,10 +179,14 @@ export function Create({ args }: { args: string[] }) {
         )
       )}
       {step === Steps.Done && (
-        <Text>{`\nTo start the project, run:
+        <Text>{`
+To start the project, run:
   cd ${targetDir}
   ${pkgManager} install
-  ${pkgManager}${pkgManager === "yarn" ? "" : " run"} dev
+  ${formatRunCommand("dev")}
+To upload your game, run:
```

Comment:
```suggestion
\nTo upload your game, run:
```

IMO line breaks help with clarity